src/_P097_Esp32Touch.ino

```diff
@@ -204,26 +207,46 @@ void P097_setEventParams(int pin, uint16_t threshold) {
       case 7: touchAttachInterrupt(T7, P097_got_T7, threshold); break;
       case 8: touchAttachInterrupt(T8, P097_got_T8, threshold); break;
       case 9: touchAttachInterrupt(T9, P097_got_T9, threshold); break;
+#ifdef ESP32S2
+/*
+      case 10: touchAttachInterrupt(T10, P097_got_T10, threshold); break;
```

Comment:
Not enabled/activated yet?

Reply:
Still trying to debug why some builds keep crashing when flashed.
It seems somewhat random whether a build is working or not.